src/BlockNumber.jsx

```diff
@@ -13,12 +13,12 @@ export default function BlockNumber(props) {
     : api.derive.chain.bestNumber;
 
   useEffect(() => {
-    let unsub = bestNumber(number => {
+    let unsubscribeAll = bestNumber(number => {
```

Comment:
This isn't right.

We need to do the `.then()` stuff. Please look at the `balances` component.